cmd/oras/root/manifest/index/create.go

```diff
@@ -204,3 +216,18 @@ func pushIndex(ctx context.Context, target oras.Target, desc ocispec.Descriptor,
 	}
 	return printer.Println("Digest:", desc.Digest)
 }
+
+func parseAnnotations(input []string) (map[string]string, error) {
```

Comment:
This part is copied and modified from `packer.go`, refactoring may be needed.

Reply:
Do we consider exporting the original `parseAnnotationFlags` function in `packer.go` and reusing it here?

Reply:
Better refactor `packer` option to isolate out `annotation` flag parsing

```mermaid
flowchart LR
    subgraph Command
	A(push)
	B(attach)
	C(manifest index create)
    end
    subgraph Option
	subgraph packer
		D[annotation]
	end
    end
    A ----> packer
    B ----> packer
    C --> D
```

You man also replace`LoadManifestAnnotations()` with `annotation.Parse`